core/store/src/trie/shard_tries.rs

```diff
@@ -67,6 +73,7 @@ impl ShardTries {
             prefetchers: Default::default(),
             state_snapshot: Arc::new(RwLock::new(None)),
             state_snapshot_config,
+            shard_uid_to_db_prefix,
```

Comment:
You have this mapping in the db and here in memory. What is your strategy for keeping them consistent?

Reply:
Switched to in memory mapping.